monsterui/core.py

```diff
@@ -182,7 +183,7 @@ def _generate_next_value_(name, start, count, last_values): return name
     violet = auto()
     zinc = auto()
 
-    def _create_headers(self, urls, mode='auto', icons=True, daisy=True, highlightjs=False, katex=True, radii=ThemeRadii.sm, shadows=ThemeShadows.sm, font=ThemeFont.sm):
+    def _create_headers(self, urls, mode='auto', icons=True, daisy=True, highlightjs=False, katex=True, charts=True, radii=ThemeRadii.sm, shadows=ThemeShadows.sm, font=ThemeFont.sm):
```

Comment:
```suggestion
    def _create_headers(self, urls, mode='auto', icons=True, daisy=True, highlightjs=False, katex=True, apex_charts=False, radii=ThemeRadii.sm, shadows=ThemeShadows.sm, font=ThemeFont.sm):
```

I'm a little torn on the param name. `charts` is more concise but `apex_charts` is more future proof. I lean slightly towards `apex_charts`.
 
Let's  add `apex_charts=False` to `headers` and `local_headers` as well. This will allow users to easily include apex headers in their app.